Thrive.csproj

```diff
@@ -71,6 +71,7 @@
     <Compile Include="src\general\StringUtils.cs" />
     <Compile Include="src\general\TranslationHelper.cs" />
     <Compile Include="src\general\TypeUtils.cs" />
+	<Compile Include="src\general\ShuffleBag.cs" />
```

Comment:
I did! Otherwise Godot would have been endlessly complaining. I didn't know there was a non manual way to do this.
(Also I wasn't able to open this file in VisualStudio).

It comes from an automatic tab indentation from my text editor, which I've fixed. Still, I'm surprised that this passed the auto-checks.

Reply:
I usually create a new C# script through the Godot editor so that way I think Godot could register the file and automatically add it into the csproj. I'm assuming you uses an external editor which in this case probably the reason why it didn't get added because it kind of bypass Godot's detection. I personally haven't tried creating new scripts externally (surprisingly) so couldn't really confirm.

Reply:
Oh, this definitely is the reason. I create my files directly through VisualStudio (as I'm not used to Godot), and indeed it is not detected before I modify the .csproj. I'll do it from Godot next time!

Reply:
I create (most) new script files with Rider, and that properly updates the csproj.